util/check_characters.py

```diff
@@ -4,27 +4,30 @@
 
 '''
 Usage: $ python3 check_characters.py "INFILE"
- e.g.  $ python3 ../DeepSpeech/util/check_characters.py "/home/data/*.csv" 
- e.g.  $ python3 ../DeepSpeech/util/check_characters.py "/home/data/french.csv" 
- e.g.  $ python3 ../DeepSpeech/util/check_characters.py "train.csv test.csv" 
+ e.g.  $ python3 check_characters.py -csv /home/data/french.csv
+ e.g.  $ python3 check_characters.py -csv ../train.csv,../test.csv 
+ e.g.  $ python3 check_characters.py -alpha -csv ../train.csv 
 
 Point this script to your transcripts, and it returns 
 to the terminal the unique set of characters in those 
 files (combined).
 
-These files are assumed to be comma delimited, 
-with the transcript being the third field.
+These files are assumed to be csv, with the transcript being the third field.
 
 The script simply reads all the text from all the files, 
 storing a set of unique characters that were seen 
 along the way.
 '''
+import argparse
+import os
 
-inFiles=sys.argv[1]
-if "*" in inFiles:
-    inFiles = glob.glob(inFiles)
-else:
-    inFiles = inFiles.split()
+parser = argparse.ArgumentParser()
+
+parser.add_argument('-csv', '--csv-files', help='Str. Filenames as a comma separated list', required=True)
+parser.add_argument("-alpha", "--alphabet-format",help="Bool. Print in format for alphabet.txt",action='store_true')
```

Comment:
Nit: mixing single and double quotes.

Reply:
I saw that after i sent the PR and briefly debated fixing it:) I'll fix it and resend and merge

Reply:
quotes resolved in second commit